src/calculateRentalCost.js

```diff
@@ -4,7 +4,9 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The function parameter name 'days' could be more descriptive. Consider using a name like 'numberOfDays' to better convey its purpose. This aligns with the checklist item: 'Make names descriptive, so they explain the purpose of variables and functions.'